sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -243,6 +243,15 @@ public enum AccountKind
         FileStorage = 3,
         BlockBlobStorage = 4,
     }
+    public partial class AppendBlobAppendBlockFromUriOptions
+    {
+        public AppendBlobAppendBlockFromUriOptions() { }
+        public Azure.Storage.Blobs.Models.AppendBlobRequestConditions DestinationConditions { get { throw null; } set { } }
+        public string SourceBearerToken { get { throw null; } set { } }
```

Comment:
Do we know if other schemes are going to be supported in the future (https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication#authentication_schemes )?